.travis.yml

```diff
@@ -1,15 +1,57 @@
 language: php
 
-php:
-  - 5.3.3
-  - 5.3
-  - 5.4
-  - 5.5
-  - 5.6
-  - hhvm
+cache:
+    directories:
+      - $HOME/.composer/cache/files
+
+before_install:
+    # Circumvent a bug in the travis HHVM image - ships with incompatible PHPUnit.
```

Comment:
I would remove everything related to HHVM as we (Symfony) do not support HHVM anymore anyway.